lib/torque/core.js

```diff
@@ -62,6 +62,7 @@
   var types = {
     Uint8Array: typeof(global['Uint8Array']) !== 'undefined' ? global.Uint8Array : Array,
     Uint32Array: typeof(global['Uint32Array']) !== 'undefined' ? global.Uint32Array : Array,
+    Int16Array: typeof(global['Int16Array']) !== 'undefined' ? global.Int16Array : Array,
```

Comment:
This is to expose the type we will use to render out of bounding box data for individual tile rendering.